src/Nethermind/Nethermind.Network.Enr/Nethermind.Network.Enr.csproj

```diff
@@ -8,12 +8,12 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\Nethermind.Crypto\Nethermind.Crypto.csproj" />
-      <ProjectReference Include="..\Nethermind.Network\Nethermind.Network.csproj" />
+      <PackageReference Include="Nethermind.DotNetty.Buffers" Version="0.7.5-preview.1" />
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="DotNetty.Buffers" Version="0.7.2" />
+      <ProjectReference Include="..\Nethermind.Crypto\Nethermind.Crypto.csproj" />
```

Comment:
is this needed here?

Reply:
I don't know. I just replaced the package. Please check, if we don't, I'll remove it.